src/types/duration.rs

```diff
@@ -0,0 +1,59 @@
+use napi::{bindgen_prelude::BigInt, Error, Status};
+use scylla::frame::value::CqlDuration;
+
+#[napi]
+pub struct DurationWrapper {
+  pub months: i32,
+  pub days: i32,
+  pub nanoseconds: i64,
+}
+
+#[napi]
+impl DurationWrapper {
+  #[napi]
+  pub fn new(months: i32, days: i32, ns_bigint: BigInt) -> napi::Result<Self> {
+    let ns = ns_bigint.get_i64();
+    if !ns.1 {
```

Comment:
It's not clear at all what is `ns.1` and `ns.0`.

Let's aid readability by using deconstruction instead of `.1` access.
```suggestion
    let (_, ns) = ns_bigint.get_i64(); // Assuming that a two-element tuple is returned
    if ns {
```